meetings/2025-10-02.md

```diff
@@ -0,0 +1,147 @@
+# Solid Practitioners
+
+## Meeting Date
+
+- 2025-10-02
+
+## Details
+
+- call https://meet.jit.si/solid-practitioners
+- chat https://matrix.to/#/#solid-practitioners:matrix.org
+- repo https://github.com/solid-contrib/practitioners
+- recordings https://spectra.video/c/solid_practitioners/videos
+
+## Facilitator
+
+- Michal
+
+## Scribe
+
+- Theo
+
+## Participants (please add yourself)
+
+- Jeff Zucker
+- [Luke Dary](https://www.linkedin.com/in/lukedary)
+- [Michal](https://id.mrkvon.org)
+- Tom Byrd
+- [Fred Gibson](https://graphmetrix.com)
+- [Theo @thhck](http://github.com/thhck)
+- [Rui Zhao](https://me.ryey.icu)
+- Marc Haddle
+- [Jesse Wright](https://jeswr.solidcommunity.net/)
+- [Wilfred Pinfold](https://www.linkedin.com/in/wilfred-pinfold-aa6b68/)
+- John Kirkwood
+- \_
+
+## Agenda
+
+### Check-in
+
+- How are you coming to the meeting? (brief, round, 3-5 min, not minuted)
+
+### Administrative
+
+- Call for Scribe
+- ~~Objections to recording session?~~ (not recorded)
+- Review agenda
+
+### New Member Check-in (brief intro, current work)
+
+- Luke: work at Redhat. Here to talk about adoption of Solid. Also volunteer with non-profit, which pay money for auth or storage, would be a lower cost for them to use Solid.
+- ?: You talk also about issue convincing ppl at Redhat to use Solid ?
+- L: There is a need to capture user data ( lightmode, language .. ), because of our ecosystem at Redhat and try to bring them together using GraphQL and I proposed Solid at a solution and the answer was "we already have system that can do what Solid can do ( auth etc...)" . I try to convice them that Solid is a better option. In there perspective if user don't request it, no need to implement it.
```

Comment:
```suggestion
- L: There is a need to capture user data ( lightmode, language .. ), because of our ecosystem at RedHat and efforts to bring them together using GraphQL. I proposed Solid at a solution, and the answer was "we already have systems that can do what Solid can do ( auth, etc...)" . I try to convince them that Solid is a better option. In their perspective, "if users don't request it, no need to implement it."
```